### DISCUSSION TITLE
RelayCommand CanExecute on command parameter

### DISCUSSION THREAD

**Discussion by DGNicolaz** [CommunityToolkit/Maui#2355] [Q&A]
Hi,
i was used to have my commands with parameters like for example an entry with its return command would have had it's entry.text as a command parameter and in my command declaration the CanExecute was an evaulation of the command parameter, if not null then command can execute.
Now i'm using the community toolkit with the relay command attribute and i don't see the possibility to do the same without binding my command parameter to the view model. Am i missing something?
Thanks

**Comment by bijington** [comment]
Sorry you've got the wrong repository. You want this one https://github.com/CommunityToolkit/dotnet
